### ISSUE TITLE
Allow to specify the path to cypress executable

### ISSUE THREAD

**Issue by rusakov92** [mbolotov/intellij-cypress#3]
Hello, my current setup in PHPStorm is with a remote PHP interpreter and there is a folder mapping to my root directory from my local PC to my docker container. Example:

In Preferences -> Languaged & Frameworks -> PHP
`Path mappings: <Project root>->/code`
Where project root is `/User/myname/code`

It looks like the plugin uses the mapped remote folder as a project root and tries to execute my cypress test by appending `/code` as the project root instead of `/User/myname/code`. Here is what I see from the runner configuration:
```
node /code/node_modules/.bin/cypress run --env headless=true --config-file cypress.json --headless run --headed --reporter /User/myname/code/node_modules/cypress-intellij-reporter --spec /User/myname/code/tests/Cypress/integration/autorization_code/authorization_code_generation.js
```

Is there a way to specify the full path to the executable?

**Comment by mbolotov** [comment]
Hi, 
No there is no way to specify the full path now. I'll think about this feature. 
Currently, you could try to run cypress via a package manager:
![image](https://user-images.githubusercontent.com/19785715/77794017-32472380-707c-11ea-9506-9611ed34dfc2.png)

Is it possible to share your project setup somehow?

**Comment by rusakov92** [comment]
Yes, those are my current settings, I usually run cypress locally because from the remote container I can only run it in headless mode but I also set up PHPStorm to use the remote interpreter because of custom packages and settings that are specific for the version in the container.

Problem is that the app uses my local node executable as intended but appends the incorrect project root dir to the cypress executable in the node_modules folder. If I'm able to specify my local custom path to cypress executable this will resolve my problem.

**Comment by mbolotov** [comment]
> those are my current settings,

Hmm, do you mean you have the npm selected and have `node /code/node_modules/.bin/cypress run` in the command?
That looks quite weird since this form of command is expected for a no package manager invocation type. 
When I set the npm I get the following:
`"C:\Program Files\nodejs\node.exe" "C:\Program Files\nodejs\node_modules\npm\bin\npx-cli.js" cypress run ...`
So there is no explicit path to cypress package in this case.

**Comment by rusakov92** [comment]
Had another check on the configuration, turns out yarn is set as a package manager.
Here is my config, note that all paths are local on my Mac.
![Screenshot 2020-03-28 at 19 33 46](https://user-images.githubusercontent.com/7600983/77829500-2ebea580-712b-11ea-8028-722691701cdf.jpg)
![Screenshot 2020-03-28 at 19 34 55](https://user-images.githubusercontent.com/7600983/77829509-3a11d100-712b-11ea-8367-4e641bc1210b.jpg)

I've set it to use NPM:
![Screenshot 2020-03-28 at 19 43 59](https://user-images.githubusercontent.com/7600983/77829700-81e52800-712c-11ea-81cc-65313402a8dd.jpg)

Works fine without any issues and executes all tests.
I guess there is no issue here then, I did not configure correctly my runner. Sorry about that.
